examples/video-chat/android/settings.gradle

```diff
@@ -5,3 +5,5 @@ includeBuild('../../../node_modules/@react-native/gradle-plugin')
 
 apply from: new File(["node", "--print", "require.resolve('expo/package.json')"].execute(null, rootDir).text.trim(), "../scripts/autolinking.gradle")
 useExpoModules()
+include ':android-client'
```

Comment:
consider more verbose name (for example that contains fishjam)
if a library user has a module named "android-client" the build will break and "android-client" doesn't tell much

Reply:
it is not the case.
1. Jitpack versoin would be under adress of `com.github.organisation.repo.(tag | commit | branch)
2. Imports in kotlin files files are determined by `android-client-sdk/FishjamClient/build.gradle` via namespace
3. `:android-client' is only use in example app, so it will not brak anything

Reply:
are you sure? when user uses our library in their app, it's autolinked similar to our example app. That means in Android Studio there will be a project named "fishjam-dev-react-native-client" which will have a build.gradle with those lines:
```
  if (findProject(':android-client')) {
    implementation project(':android-client')
```
This gradle file will be run when user builds his app. Now if in the user's app there is a project named "android-client" (I know it's really unlikely, but it's not nice to use such a generic name), then it'll be included as a dependency here instead of a remote library.

Reply:
Ahh, I see your point

Reply:
my proposition is then `fishjam-cloud-android-client`

Reply:
ok 👍